File: backend/src/main/java/cha_revelacao/dto/request/GaleriaPostRequest.java
package cha_revelacao.dto.request;

import jakarta.validation.constraints.NotNull;
import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class GaleriaPostRequest {

    @Size(max = 5000, message = "Mensagem deve ter no máximo 5000 caracteres")
    private String mensagem;

    private String urlFoto;
    
    private String urlVideo;
    
    @NotNull(message = "ID do evento é obrigatório")
    private Long eventoId;
}

File: backend/src/DTOs/Requests/GaleriaPostRequest.cs
namespace MemuVie.Evento.DTOs.Requests;

public class GaleriaPostRequest
{
    public string? Mensagem { get; set; }
    public string? UrlFoto { get; set; }
    public string? UrlVideo { get; set; }
    public long EventoId { get; set; }
}
